tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -42,3 +44,69 @@ Feature: spaces management
 #      | id     |
 #      | team.b |
 #    And "Brian" logs out
+  Scenario: multiple spaces can be managed at once in the admin settings
```

Comment:
Please add 2 lines above. Scenarios should always be separated by two lines. One of them is reserved for annotations like `@skip`.